.github/paths-labeller.yml

```diff
@@ -0,0 +1,53 @@
+---
+- "beats":
+  - "detection_rules/beats.py"
+  - "etc/beats_schemas/**/*"
+- "documentation":
+  - "./**/*.md"
+- "Domain: Cloud":
+  - "rules/integrations/aws/**/*.toml"
+  - "rules/integrations/azure/**/*.toml"
+  - "rules/integrations/gcp/**/*.toml"
+  - "rules/integrations/google-workspace/**/*.toml"
+  - "rules/integrations/microsoft-365/**/*.toml"
+  - "rules/integrations/okta/**/*.toml"
+- "Domain: Endpoint":
+  - "rules/windows/**/*.toml"
+  - "rules/linux/**/*.toml"
+  - "rules/macos/**/*.toml"
+- "ecs":
+  - "detection_rules/ecs.py"
+  - "etc/ecs_schemas/**/*"
+- "ML":
+  - "rules/ml/**/*"
+- "OS: Linux":
+  - "rules/linux/**/*.toml"
+- "OS: macOS":
+  - "rules/macos/**/*.toml"
+- "OS: Windows":
+  - "rules/windows/**/*.toml"
+- "Platform: AWS":
```

Comment:
Think we should rename these labels from Platform to Integration?
Seems logical and like a good time to do it.

Reply:
sounds good - changing now